### DISCUSSION TITLE
Backlog #3

### DISCUSSION THREAD

**Discussion by Sinan-Karakaya** [Sinan-Karakaya/R-Type#69] [Announcements]
## What we have done:

@Sinan-Karakaya:

- Finished MVP for the editor
- Started helping @jovanHillion 

@JasonLAGOUTE:

- Packet implementation
- Server boilerplate

@lucasdpt:

- Implementation of network lib
- Small features of server

@mlbonniec:

- Finished UIRectangleElement
- Cried on MacOS

@jovanHillion:

- WIP Lua implementation

## What we are going to do

@Sinan-Karakaya:

- Finish the scripting and MVP

@JasonLAGOUTE & @lucasdpt:

- Ending the implementation of the server for the MVP
- Finish the implementation of the network lib

@mlbonniec:

- Implement network on client side

@jovanHillion:

- Finish Lua implementation and finish MVP

**Comment by mlbonniec** [comment]
### To-do MVP

**All**
- Documentation

**Iencli**
- Intégrer la librairie network (statique)
- Capter les événements et appeler le runtime

**Runtime**
- Lua à implémenter
- Gérer les événements
- Minor bug fixes (ex. scale render window)

**Editor**
- Mettre à jour avec le runtime
- Fixer le segfault de fermeture
- Play / Pause (not for MVP)

**Server**
- Mettre à jour avec le runtime
- Terminer l’implémentation de la librairie network
- Terminer l’implémentation de la logique de synchronisation client
